README.md

```diff
@@ -44,6 +44,9 @@ only be available if you provide configuration respectively for them.
               api_key: <string_android_gcm_api_key> # This is titled "Server Key" when creating it
               use_multi_curl: <boolean_android_gcm_use_multi_curl> # default is true
               dry_run: <bool_use_gcm_dry_run>
+          fcm:
+              api_key: <string_android_fcm_api_key> # This is titled "Server Key" when creating it
```

Comment:
I'm not that familiar with FCM - is the comment still relevant here about "Server Key" being how the Firebase control panels name the API key?